### ISSUE TITLE
question on Xconv implementation

### ISSUE THREAD

**Issue by xiaolinpan** [rusty1s/pytorch_geometric#1042]
## ❓ Questions & Help
for Xconv, if i have features metrics in shape [1024, 3], 
     if x is not None:
            x = x.unsqueeze(-1) if x.dim() == 1 else x
            x = x[col].view(N, K, self.in_channels) ??? [4096,  3] --> [1024, 4, 1024]
            x_star = torch.cat([x_star, x], dim=-1)
why the shape can't adapt ?  Thank you!

**Comment by rusty1s** [comment]
I am not totally sure what you mean here. Is `in_channels` set to `3`?

**Comment by xiaolinpan** [comment]
`in_channels` is not the 1024? 

> in_channels (int) – Size of each input sample.

could you give an example about xconv? the dataset include `x` and `pos`, thank you vary much?
